src/test/java/io/github/mfvanek/pg/utils/DatabasePopulator.java

```diff
@@ -313,13 +313,13 @@ private void createTableWithoutPrimaryKey() {
             statement.execute(String.format("create table if not exists %s.bad_clients (" +
                     "id bigint not null, " +
                     "name varchar(255) not null)", schemaName));
-            final String checkQuery = String.format("select exists (\n" +
-                    "   select 1 \n" +
-                    "   from pg_catalog.pg_class c\n" +
-                    "   join pg_catalog.pg_namespace n on n.oid = c.relnamespace\n" +
-                    "   where n.nspname = '%s'\n" +
-                    "   and c.relname = '%s'\n" +
-                    "   and c.relkind = 'r'\n" +
+            final String checkQuery = String.format("select exists (%n" +
```

Comment:
Something strange here. Please revert these changes

Reply:
This is SpotBugs bug VA_FORMAT_STRING_USES_NEWLINE.  "%n" always outputs the correct platform-specific line separator, it is portable across platforms whereas"\n" is not, see https://stackoverflow.com/a/1883364/4182275.